pkg/mtls/sds/sdsclient.go

```diff
@@ -41,27 +42,27 @@ var sdsPostCallback func() = nil
 var ErrSdsClientNotInit = errors.New("sds client not init")
 
 // NewSdsClientSingleton use by tls module , when get sds config from xds
-func NewSdsClientSingleton(config *auth.SdsSecretConfig) types.SdsClient {
+func NewSdsClientSingleton(config *envoy_extensions_transport_sockets_tls_v3.SdsSecretConfig) types.SdsClient {
```

Comment:
sds client should be compatible with V2 and V3, that is, we need create a new v3 of sds client.

Reply:
use `NewSdsClientSingletonDeprecated` represent V2